### ISSUE TITLE
Feature request: Socket wrench walls with pipes/wires on them to access them

### ISSUE THREAD

**Issue by kilozombie** [d3athrow/vgstation13#28504]
Would make it less of a faux pas to have walls with pipes/wires on them.

It'd basically work by just making pipes/wires visible on the topmost layer and adding a little overlay on top of the wall to make it look like a panel's been opened. Wouldn't allow you to bypass the wall to get through easier, only for pipe placing or rewiring.

**Comment by DrSnips** [comment]
I agree that this would be a really nice thing to have. Maybe it moves everything to the top/bottom allowing you to reveal or [re]hide pipes you place on top or under respectively.
